importer/importer_utils.py

```diff
@@ -439,3 +438,8 @@ def create_wlm_url(country, language, id):
     url_base = "https://tools.wmflabs.org/heritage/api/api.php?action=search&format=json&srcountry={}&srlanguage={}&srid={}"
     return url_base.format(
         country, language, quote(id))
+
+
+def create_site_instance(language, family):
```

Comment:
So there is a sneaky thing we can do here which might make life a bit easier. We make this function cache any created sites.

If we at the top of utils introduces `site_cache = {}`
then here we can use.
```python
site_key = (language, family)
site = site_cache.get(site_key)
if not site:
    site = pywikibot.Site(language, family)
    site_cache[site_key] = site
return site
```